git/libgit2/env.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# This script contains setup variables related to libgit2.
+# It's purpose is to reduce boilerplate, and provide a single source
+# of info for all other build scripts.
+
+LIBGIT2_IMG=ghcr.io/fluxcd/golang-with-libgit2
+LIBGIT2_TAG=v0.2.0
+LIBGIT2_PATH=${BUILD_DIR}/libgit2/${LIBGIT2_TAG}
+LIBGIT2_LIB_PATH=${LIBGIT2_PATH}/lib
+LIBGIT2=${LIBGIT2_LIB_PATH}/libgit2.a
+PKG_CONFIG_PATH=${LIBGIT2_LIB_PATH}/pkgconfig
```

Comment:
I think we should either have environment variables set in a `.env` file, or export such vars in a `.sh`. I am leaning more towards the latter.